packages/zcli-apps/src/commands/apps/new.ts

```diff
@@ -51,16 +53,22 @@ export default class New extends Command {
     })
   }
 
-  getScaffoldDir (flagScaffold: string): string {
-    const scaffolds: Scaffolds = {
-      basic: 'apps_scaffold_basic',
-      react: 'app_scaffold'
-    }
-    const scaffoldRepo = scaffolds[flagScaffold]
-    if (!scaffoldRepo) {
-      throw new CLIError(chalk.red(`Invalid scaffold option entered ${flagScaffold}`))
-    }
-    return scaffoldRepo
+  async extractPackageIfExists (flagScaffold: string) {
+    return new Promise((resolve, reject) => {
+      fsExtra.copy(
+        path.join(process.cwd(), '/', 'app_scaffolds-master/packages/', flagScaffold),
+        path.join(process.cwd(), '/', `app_scaffolds-master-${flagScaffold}`),
+        { overwrite: true, errorOnExist: true }, (err: FsExtraError) => {
+          if (err) {
+            if (err.code === 'ENOENT') {
+              reject(new Error(`Package ${flagScaffold} does not exist: ${err}`))
```

Comment:
I think this should be called scaffold, package is just a folder where we keep scaffolds, so this error may not make sense for 3rd party devs.

Probably the function name should also be called `extractScaffoldIfExists` for readability?